server/src/services/library.service.spec.ts

```diff
@@ -296,33 +223,48 @@ describe(LibraryService.name, () => {
     });
   });
 
-  describe('handleOfflineCheck', () => {
-    it('should skip missing assets', async () => {
-      const mockAssetJob: ILibraryOfflineJob = {
-        id: assetStub.external.id,
-        importPaths: ['/'],
-        exclusionPatterns: [],
-      };
+  describe('handleQueueRemoveDeleted', () => {
+    it('should queue online check of existing assets', async () => {
+      libraryMock.get.mockResolvedValue(libraryStub.externalLibrary1);
+      storageMock.walk.mockImplementation(async function* generator() {});
```

Comment:
Can't you just `.mockImplementation(() => {})`?

Reply:
No, it didn't work out like that